scipy/stats/tests/test_distributions.py

```diff
@@ -3140,6 +3140,7 @@ def test_endpoints(self):
         a, b = 0.2, 3
         assert_equal(stats.beta.pdf(0, a, b), np.inf)
 
+    @pytest.mark.xfail(IS_PYPY, reason="Does not convert boost warning")
     def test_boost_eval_issue_14606(self):
```

Comment:
We can take this out of course. At the time, Matti [commented](https://github.com/conda-forge/scipy-feedstock/pull/196#issuecomment-979317832):

> [That test](https://github.com/scipy/scipy/blob/1ed13610f7d868441cccc76072a9659c49f3cf4c/scipy/stats/tests/test_distributions.py#L2901) is for [converting a boostpython error](https://github.com/scipy/scipy/pull/14618) into a warning. Apparently PyPy does not hit the code path to create the boost-python warning in the first place. I would suggest here too skipping the test on PyPy at least until SciPy expressly supports PyPy.
> 
> ```
> with pytest.warns(RuntimeWarning):
>     stats.beta.ppf(q, a, b)
> ```

I think pypy is much more expressly supported now, so perhaps this is worth revisiting.